.changeset/soft-crews-melt.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hono/vite-dev-server': patch
```

Comment:
This is a `chore` PR. But we need to publish it on the npm. So, I added the changeset.